### PR TITLE
New questions to GitHub Actions

### PR DESCRIPTION
**Pull request by dostonhamrakulov**
## PR Type

- [x] Adding new question(s)
> **Warning**: We do not support the inclusion of questions directly copied from official GitHub certification exams. Please only submit original questions and content that you have created.
- [ ] Other content changes (updating questions, answers, explanations or study resources)
- [ ] Code changes (non-content)
- [ ] Documentation changes
- [ ] Other

## What's new?
New questions to GitHub Actions

### CODE REVIEW

**Comment by FidelusAleksander** [comment]
Hi @dostonhamrakulov please update your branch with latest changes from master

**Comment by dostonhamrakulov** [comment]
> Hi @dostonhamrakulov please update your branch with latest changes from master

I have updated

File: content/questions/actions/question-109.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+question: "What is the purpose of the `timeout-minutes` keyword in a step?"
+archetype: "questions"
+title: "Question 109"
+draft: false
+---
+
+
+> https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idtimeout-minutes
```
Comment (FidelusAleksander) [inline content/questions/actions/question-109.md:9]: This is the one for steps, you linked the section for job.timeout-minutes
```suggestion
> https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepstimeout-minutes
```

File: content/questions/actions/question-109.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+1. [x] it limits the execution time for individual commands within a step
```
Comment (FidelusAleksander) [inline content/questions/actions/question-109.md:10]: I can't find any source that it limites the execution time for "individual commands" within a step. 

All I see is that it limits the execution time of the **entire** step.

Do you have some source for that?

**Review by FidelusAleksander — CHANGES_REQUESTED** [review]

File: content/questions/actions/question-109.md (hunk 2 above)
Comment (FidelusAleksander) [reply content/questions/actions/question-109.md:10]: Steps can have [multi line commands ](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsrun) and I'm pretty sure that the `timeout-minutes` is not for individual commands but for the step as a whole.

Comment (dostonhamrakulov) [reply content/questions/actions/question-109.md]: Yes, you are right, I have formulated it wrongly. I have updated the answer

File: content/questions/actions/question-109.md (hunk 1 above)
Comment (dostonhamrakulov) [reply content/questions/actions/question-109.md]: updated

**Review by FidelusAleksander — APPROVED** [review]
LGTM!

Thanks for contributing again! 🚀

**Pull request merged**